environ.py

```diff
@@ -1,25 +1,28 @@
 # standard libraries
-import json, logging, os
+import json
+import logging
+import os
 from json.decoder import JSONDecodeError
+from typing import Union
 
+# Set up ENV
 # entry-level job modules need to be one-level beneath root
-ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+ROOT_DIR: str = os.path.dirname(os.path.abspath(__file__))
+CONFIG_DIR: str = os.path.join(ROOT_DIR, os.getenv('ENV_DIR', os.path.join('config', 'secrets')))
+CONFIG_PATH: str = os.path.join(CONFIG_DIR, os.getenv('ENV_FILE', 'env.json'))
```

Comment:
This looks like a fine solution to the problem we were discussing earlier. We'll need to update the README's OpenShift section, since `ENV_PATH` is no longer a thing.

Reply:
✅Fixed.